bundles/org.openhab.binding.warmup/README.md

```diff
@@ -50,15 +48,15 @@ Once credentials are successfully added to the bridge, any rooms (devices) detec
 
 ## Thing Configuration
 
-### My Warmup Account
+### `my-warmup` Bridge Configuration
 
 | config parameter | type    | description                                     | required | default |
 |------------------|---------|-------------------------------------------------|----------|---------|
 | username         | String  | Username for my.warmup.com                      | true     |         |
 | password         | String  | Password for my.warmup.com                      | true     |         |
 | refreshInterval  | Integer | Interval in seconds between automatic refreshes | true     | 300     |
 
-### Room
+### `Room` Thing Configuration
```

Comment:
Maybe rather room all in lowercase as it is the real thing type ?

Reply:
Yes, thanks!